web/src/components/pages/kiosk/kiosk.css

```diff
@@ -0,0 +1,46 @@
+@media (--mobile) {
+    #common-voice-kiosk {
+        top: 5.5rem;
+        width: 100%;
+        text-align: center;
+
+        & h1 {
+            display: flex;
+            justify-content: center;
+            margin: 0 auto;
+            width: auto;
+        }
+    }
+}
+
+@media (--desktop) {
+    #common-voice-kiosk {
+        top: 10rem;
+        text-align: end;
+    }
+}
+
+#kiosk-container {
+    & .kiosk a {
+        font-size: var(--font-size-small);
+        font-weight: 600;
+        text-decoration: underline;
+    }
+}
+
+#common-voice-kiosk {
+    & {
```

Comment:
Haven't seen this `& {` kind of rule declaration before. Is there a difference to directly putting the rules under `#common-voice-kiosk`?